### PR TITLE
improve mia description in readme

### LINKED ISSUE #115: Improve MIA descriptions in README
after receiving Franziskas Feedback after the current iteration

### PR DESCRIPTION
**Pull request by Friedrich-Mueller**
Closes #115.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -51,6 +51,53 @@ be arbitrarily chosen.
...
+
+Once the meta-classifier is trained, it can be used to infer membership status on an array of one or more unknown unlabeled data records _X<sub>unkown</sub>_, returning for each record either  a 1, indicating a member, or a 0, indicating a non-member of the target model’s training set _(X,y)<sub>target</sub>_.
+
+The current implementation gives the user the choice of the attack model meta-classifier. The default is a neural network, but random forests and gradient boosting models can be chosen as well.
+
+![](docs/mia_blackbox.png)
```
Comment (Erdnaf) [inline README.md:81]: this image does have a transparent background but the others have not. It is only noticable in Github dark mode, but maybe a small change

File: README.md [hunk 2]
```diff
@@ -51,6 +51,53 @@ be arbitrarily chosen.
...
+
+Once the shadow model is trained, it can be used to infer membership status on an array of one or more unknown unlabeled data records _X<sub>unkown</sub>_, returning for each record either  a 1, indicating a member, or a 0, indicating a non-member of the target model’s training set _(X,y)<sub>target</sub>_.
+
+![](docs/mia_blackbox_decision_boundary.png)
+
+#### Membership Inference Black-Box Rule-Based
```
Comment (Erdnaf) [inline README.md:93]: For me the `####` formatig is barely noticable over normal text. Maybe this can be restructured that we only are using up to `###`?

**Review by Erdnaf — APPROVED** [review]
Good work. Much more redable than the other one. Found small things but nothing to urgent.

i think this schould be rebased to `team1sprint4` right?

**Comment by Friedrich-Mueller** [comment]
> 
> 
> Good work. Much more redable than the other one. Found small things but nothing to urgent.
> 
> i think this schould be rebased to `team1sprint4` right?

**Comment by Friedrich-Mueller** [comment]
> 
> 
> Good work. Much more redable than the other one. Found small things but nothing to urgent.
> 
> i think this schould be rebased to `team1sprint4` right?

Oh, yeah definately.

File: README.md (hunk 1 above)
Comment (jtorhoff) [reply README.md:81]: I've changed all images to have white background and also higher resolution.

**Pull request merged**